frontend/src/editor/actions/useCellActionButton.tsx

```diff
@@ -57,6 +62,16 @@ export function useCellActionButtons({
   // Actions
   const actions: ActionButton[][] = [
     [
+      {
+        icon: <PlayIcon size={13} strokeWidth={1.5} />,
+        label: "Run cell",
+        hotkey: "cell.run",
+        hidden:
+          status === "running" ||
+          status === "disabled-transitively" ||
```

Comment:
Should we also add `status === "queued" || ` here?

Reply:
yep, done